internal/pkg/vm/actor/builtin/default.go

```diff
@@ -21,5 +22,6 @@ var DefaultActors = dispatch.NewBuilder().
 	Add(specs.MultisigActorCodeID, &multisig.Actor{}).
 	Add(specs.PaymentChannelActorCodeID, &paych.Actor{}).
 	Add(specs.StoragePowerActorCodeID, &power.Actor{}).
+	Add(specs.StorageMarketActorCodeID, &market.Actor{}).
```

Comment:
Eek, could you please add all the actors for specs-actors builtin/codes.go. There are 10 total. We can fix the debt that this is likely to rot later.

Reply:
great idea